docs/core/preview3/tools/dotnet-new.md

```diff
@@ -15,45 +15,87 @@ ms.assetid: fcc3ed2e-9265-4d50-b59e-dc2e5c190b34
 #dotnet-new (Tooling RC3)
 
 > [!WARNING]
-> This topic applies to Visual Studio 2017 RC - .NET Core Tools RC3. For the .NET Core Tools Preview 2 version,
+> This topic applies to Visual Studio 2017 RC - .NET Core Tools RC4. For the .NET Core Tools Preview 2 version,
 > see the [dotnet-new](../../tools/dotnet-new.md) topic.
 
 ## Name
-dotnet-new -- Creates a new .NET Core project in the current directory.
+dotnet-new - Creates a new .NET Core project in the current directory
 
 ## Synopsis
-`dotnet new [--help] [--type] [--lang]`
+`dotnet new [arguments] [options]`
 
 ## Description
 The `dotnet new` command provides a convenient way to initialize a valid .NET Core project and sample source code to try out the Command Line Interface (CLI) toolset. 
 
-This command is invoked in the context of a directory. When invoked, the command will result in two main artifacts being dropped to the current directory: 
-
-1. A `Program.cs` (or `Program.fs`) file that contains a sample "Hello World" program.
-2. A valid csproj project file.
+This command is invoked in the context of a directory. When invoked, the command will scaffold out the resources and files according to the template and options passed into the command. 
 
 After this, the project is ready to be compiled and/or edited further. 
 
```

Comment:
the new templates that are available come from the templating system and are subject to change. I'd rather put in the command to see  the templates for a specific type.

i.e. `dotnet mvc -list`